src/libs/actions/Policy/Policy.ts

```diff
@@ -4505,7 +4505,7 @@ function updateInvoiceCompanyWebsite(policyID: string, companyWebsite: string) {
     const parameters: UpdateInvoiceCompanyWebsiteParams = {
         authToken,
         policyID,
-        companyWebsite,
+        companyWebsite: Str.sanitizeURL(companyWebsite, CONST.COMPANY_WEBSITE_DEFAULT_SCHEME),
```

Comment:
We don't need this, we can pass the sanitized `companyWebsite` value as I explained it [here](https://github.com/Expensify/App/pull/51545/files#r1825041773)